src/config/dynamic-theme-fixes.config

```diff
@@ -7509,6 +7508,12 @@ CSS
 .buj {
     background-image: url(//ssl.gstatic.com/ui/v1/icons/mail/rfr/density_compact_v1_1x.png) !important;
 }
+div[class*="bym"][role="navigation"] {
+    background-color: rgb(25 27 28) !important;
```

Comment:
Could we something that isn't hardcoded?

Reply:
Fixed.